src/EnergyPlus/Autosizing/CoolingAirFlowSizing.cc

```diff
@@ -80,7 +80,7 @@ Real64 CoolingAirFlowSizer::size(EnergyPlusData &state, Real64 _originalValue, b
             } else if (this->zoneEqSizing(this->curZoneEqNum).DesignSizeFromParent) {
                 this->autoSizedValue = this->zoneEqSizing(this->curZoneEqNum).AirVolFlow;
             } else {
-                auto const SELECT_CASE_var(this->zoneEqSizing(this->curZoneEqNum).SizingMethod(DataHVACGlobals::CoolingAirflowSizing));
+                auto const SELECT_CASE_var(this->zoneEqSizing(this->curZoneEqNum).SizingMethod(static_cast<int>(AutoSizingType::CoolingAirFlowSizing)));
```

Comment:
See comment in slack about making these plain `enum`s and doing away with `static_cast<int>`.

Reply:
Will do.